### PR TITLE
mpvScripts.modernx-zydezu: 0.2.9 -> 0.3.4

### PR DESCRIPTION
**Pull request by luftmensch-luftmensch**
## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by luftmensch-luftmensch** [comment]
```
❯ nix build .#mpvScripts.modernx-zydezu
❯ tree result/
 result
└──  share
   ├──  fonts
   │  ├──  Material-Design-Iconic-Font.ttf
   │  └──  Material-Design-Iconic-Round.ttf
   └──  mpv
      └──  scripts
         └──  modernx.lua
```

File: pkgs/applications/video/mpv/scripts/modernx-zydezu.nix [hunk 1]
```diff
@@ -7,14 +7,14 @@
...
-  version = "0.2.9";
+  version = "0.3.2";
 
   scriptPath = "modernx.lua";
   src = fetchFromGitHub {
     owner = "zydezu";
     repo = "ModernX";
     rev = finalAttrs.version;
-    hash = "sha256-Tq/v8EbojvOSSwbu8kdbLVxZWZNi1UH6zAPTcsUcgog";
+    hash = "sha256-iFIkYdTWxKO/UnN187p12/VeKHcGdPw5IN5sr7n1vHo=";
```
Comment (getchoo) [inline pkgs/applications/video/mpv/scripts/modernx-zydezu.nix:17]:
```suggestion
    hash = "sha256-RMvIh2cJrwAuWB+uOZ/JkEHyn7PjimlUZPeUb1+C118=";
```
hash mismatch on aarch64-linux. this tag might have been force pushed

**Review by getchoo — CHANGES_REQUESTED** [review]

File: pkgs/applications/video/mpv/scripts/modernx-zydezu.nix (hunk 1 above)
Comment (luftmensch-luftmensch) [reply pkgs/applications/video/mpv/scripts/modernx-zydezu.nix:17]: Didn't test it for `aarch64-linux`, but if you tested it with the new hash it's ok to me.

@getchoo  Should I commit the suggestion or squash it to the initial commit?

**Review by Guanran928 — COMMENTED** [review]
Works fine on 'x86_64-linux` after manually changing hash.

Tested via `nix build --expr 'with import ./. {}; (wrapMpv mpv-unwrapped {scripts = [mpvScripts.modernx-zydezu];})' --impure`

**Comment by fin444** [comment]
https://github.com/zydezu/ModernX/releases/tag/0.3.3

**Comment by luftmensch-luftmensch** [comment]
@Guanran928 I'll update both hash (as suggested by @getchoo) and bumped version to the 0.3.4

**Review by Guanran928 — APPROVED** [review]

**Comment by luftmensch-luftmensch** [comment]
@getchoo @Guanran928  Can we merge this if it's okay?

**Comment by luftmensch-luftmensch** [comment]
> I don't have merge permission and the merge bot currently only accepts PRs from r-ryantm

Didn't know that

**Review by nicoonoclaste — APPROVED** [review]

**Comment by luftmensch-luftmensch** [comment]
> @luftmensch-luftmensch Normally, adding yourself to `maintainer-list.nix` would be done as a separate commit. I'll split this before merging, don't worry.

@nbraud Oh I didn't know that.  Thanks!

**Pull request merged**